docs/tre-developers/end-to-end-tests.md

```diff
@@ -5,37 +5,22 @@
 1. Authentication and Authorization configuration set up as noted [here](../tre-admins/auth.md)
 1. An Azure Tre deployed environment.
 
+## Registering bundles to run End-to-end tests
+
+End-to-end tests depend on certain bundles to be registered within the TRE API.
+
+When End-to-end tests run in CI, they are registered as a prerequisite to running tests.
+
+When running tests locally, use the `prepare-for-e2e` Makefile target:
+
+```cmd
+make prepare-for-e2e
+```
+
 ## Debugging the End-to-End tests
 
-Use the 'Run and Debug' icon within Visual Studio Code and chose "E2E Extended" in the drop down box and click play.
+Use the "Run and Debug" panel within Visual Studio Code, select "E2E Extended", "E2E Smoke" or "E2E Performance" in the drop down box and click play.
 
 - This will copy /workspaces/AzureTRE/templates/core/.env to /workspaces/AzureTRE/e2e_tests/.env for you which supplies your authentciation details
 
 - This will also use /workspaces/AzureTRE/templates/core/private.env file for other values.
-
-## Running the End-to-End tests locally
-
-1. Navigate to the `e2e_tests` folder: `cd e2e_tests`
-1. Define the following environment variables:
-
-    | Environment variable name | Description | Example value |
```

Comment:
I wonder whether we have a page that describes all the environment variables - seems a shame to delete it completely. I get that it doesn't need to be here as it gets generated for you.

Reply:
We do have a page here: https://microsoft.github.io/AzureTRE/tre-admins/environment-variables/